src/plugins/mf_priority.cpp

```diff
@@ -947,19 +947,30 @@ static int validate_cb (flux_plugin_t *p,
         return flux_jobtap_reject_job (p, args, "user/bank entry has been "
```

Comment:
This commit could maybe be merged with d50f8bf1, I think it actually would have made reviewing a bit easier.